setup.py

```diff
@@ -23,6 +23,8 @@
         'keras>=2.8',
         'tensorflow>=2.9.0,<2.10', # TODO(jrute): Try to upgrade to >=2.10
         'tensorflow_gnn>=0.2.0,<0.3',  # TODO(jrute): Try to upgrade to >=0.3
+        'tf-models-official',
+        'keras-nlp',
```

Comment:
Note to self.  Remove this.  (Left over from trying premade beam search implementations.)

Reply:
Fixed.